pages/tidal tools/analyze_database.md

```diff
@@ -65,7 +65,7 @@ For the most part, all supported Database engines share very similar configurati
 
 Your YAML configuration file for Postgres, MySQL, and SQL Server must contain the following information.
 
-  - `id` - The id of the database from your Tidal account. You can find it in the URL bar when looking at a database instance. ex. If you are viewing a database instance in Tidal, the URL will show https://demo2.tidal.cloud/#/database_instances/111 in this case 111 is the database instance ID.
+  - `id` - The id of the database from your Tidal account. You can find it in the URL bar when looking at a database instance. ex. If you are viewing a database instance in Tidal, the URL will show https://[your workspace].tidal.cloud/database_instances/111/overview in this case 111 is the database instance ID.
```

Comment:
Do we have to mention the default tab name `overview` in all the links since the redirection from `/111` to `/111/overview` is already in place?

Reply:
Hmm good question. I'm mentioning it because the sentence leading up to it says that this is what the URL should look like when you're on that page. I don't think the redirect enters into it here since it's not really a link, it's just indicating to the user where the id will be, and in this case it's not at the end of the url but rather sandwiched in before `overview`.

Reply:
Makes sense in that case. 💯